dev3/2024-07-02/example-metadata/minimal.json

```diff
@@ -0,0 +1,62 @@
+{
+    "@context": [
+        "https://w3id.org/ro/crate/1.1/context",
+        {
+            "organism_classification": "https://schema.org/taxonomicRange",
+            "BioChemEntity": "https://schema.org/BioChemEntity",
+            "obo": "http://purl.obolibrary.org/obo/",
+            "acquisiton_method": {
+                "@reverse": "https://schema.org/result",
+                "@type": "@id"
+            },
+            "biological_entity": "https://schema.org/about"
+        }
+    ],
+    "@graph": [
+        {
+            "@type": "CreativeWork",
+            "@id": "ro-crate-metadata.json",
+            "conformsTo": {
+                "@id": "https://w3id.org/ro/crate/1.1"
+            },
+            "about": {
+                "@id": "./"
+            }
+        },
+        {
+            "@id": "./",
+            "@type": "Dataset",
+            "name": "Light microscopy photo of a fly",
+            "description": "Light microscopy photo of a fruit fly.",
+            "licence": "https://creativecommons.org/licenses/by/4.0/",
+            "hasPart": {
+                "@id": "./dros-mel-image.zarr/"
+            }
+        },
+        {
+            "@id": "./dros-mel-image.zarr/",
+            "@type": "Dataset",
+            "name": "OME-ZARR files",
+            "description": "the ome zarr files of the fly.",
+            "acquisiton_method": [
+                "_:b0"
+            ],
+            "preparation_method": [
+                "_b:1"
+            ]
+        },
+        {
+            "@id": "_:b0",
+            "@type": [
```

Comment:
I'm left wondering if we couldn't come up with a more readable way to make use of the ontology classes. Are there other idioms that you considred, @sherwoodf?

Reply:
This is a little tied up with using flattened form. I was considering having a property (that isn't rdf:type, but again, we'd probably need to define this) to link to these objects & then could neatly include a name field in that object alongside the ID, but with flattened this would be a whole other node in the list of objects.

Reply:
as @sherwoodf says, this would normally be handled as a separate node with its own name (and description) for human comprehensibility.

You could look at how the [Process Run Crate](https://www.researchobject.org/workflow-run-crate/profiles/process_run_crate/) uses `CreateAction` and `instrument`, but I can see the limitations of it when it comes to describing and categorising physical experimental methods rather than computational processes. 
To me it seems as if both `acquisition_method` and `preparation_method` could potentially be Process Run Crates of their own accord, if you wanted to track all the provenance of how those were carried out (or follow a similar structure within this larger crate).